assets/src/edit-story/components/form/media.js

```diff
@@ -199,6 +199,7 @@ MediaInput.defaultProps = {
   type: 'image',
   buttonInsertText: __('Choose an image', 'web-stories'),
   title: __('Choose an image', 'web-stories'),
+  ariaLabel: __('Choose an image', 'web-stories'),
```

Comment:
`defaultProps` shouldn't really be used if a prop is _required_, no? 🤔

Reply:
My latest change aligns ariaLabel with other props title, buttonInsertText (not required - have defaults).

Reply:
Now with 0d4bab414caa264dfa2f591bc07fdacb33928fed it makes  more sense 🙂